core/drivers/sub.mk

```diff
@@ -77,6 +77,7 @@ srcs-$(CFG_VERSAL_PUF) += versal_puf.c
 srcs-$(CFG_VERSAL_HUK) += versal_huk.c
 srcs-$(CFG_CBMEM_CONSOLE) += cbmem_console.c
 srcs-$(CFG_RISCV_PLIC) += plic.c
+srcs-$(CFG_HISILICON_PLAT_ACC) += hisi_trng.c
```

Comment:
This name (`CFG_HISILICON_PLAT_ACC`) is not specific enough. A few questions to help select a better name:
- We have `CFG_HI16xx_RNG` for `plat-d02` already, do you need a really new driver for `plat-d06`?
- If yes, could a similar naming scheme be used for both drivers? What are the official SoC names for D02 and D06? I believe D02 is HIP05 and D06 is HIP08. So how about `CFG_HIP05_RNG` and `CFG_HIP08_RNG`?